Firmware/Marlin_main.cpp

```diff
@@ -7925,7 +7925,7 @@ SERIAL_PROTOCOLPGM("\n\n");
     case 862: // M862: print checking
           float nDummy;
           uint8_t nCommand;
-          nCommand=(uint8_t)(modff(code_value_float(),&nDummy)*10.0+0.5);
+          nCommand=(uint8_t)(modff(code_value(),&nDummy)*10.0+0.5);
```

Comment:
we could save even more by not using `modff` ... in this case it is only here to detect `862.something`

Reply:
I have a separate PR for that :D https://github.com/prusa3d/Prusa-Firmware/pull/3795